client/lib/user/store.js

```diff
@@ -15,3 +15,11 @@ export async function clearStore() {
 	store.clearAll();
 	await clearStorage();
 }
+
+export async function getUserId() {
+	return await store.get( 'wpcom_user_id' );
```

Comment:
`store.set` and `store.get` are synchronous and don't return promises. Callers of `getUserId` work with the value as if it was synchronous, but now they start to receive promises.

Reply:
Addressed in 9cbfd723f8c1072acdac4450ea1ba3a49845cd2d.